src/translators/timescale.py

```diff
@@ -114,9 +114,12 @@ def dispose(self):
 
     def sql_error_handler(self, exception):
         analyzer = PostgresErrorAnalyzer(exception)
-        if analyzer.is_transient_error():
+        err_msg = analyzer.is_transient_error()
+        if err_msg == "ConnectionError":
             self.ccm.reset_connection('timescale')
             self.setup()
+        logging.error(err_msg)
```

Comment:
see comment above about keeping original logic of transient_error

Reply:
done